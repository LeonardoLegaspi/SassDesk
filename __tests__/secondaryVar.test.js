var sass = require('node-sass');

describe('-----SECONDARY VARIABLES TEST', ()=>{
    test('Secondary Variables snapshot', async () => {
  
        var result = sass.renderSync({
          data: `
          @import './1_helpers/secondary_variables';
          `,
          outputStyle: 'nested'
        });
        
        expect(result.css.toString()).toMatchSnapshot();
      });
  })