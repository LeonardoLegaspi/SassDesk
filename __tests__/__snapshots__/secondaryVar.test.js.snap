// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`-----SECONDARY VARIABLES TEST Secondary Variables snapshot 1`] = `""`;
